src/clj/rems/db/catalogue.clj

```diff
@@ -46,5 +46,6 @@
     return))
 
 (defn update-catalogue-item! [command]
+  ;; TODO disallow unarchiving catalogue item if its resource, form or licenses are archived
```

Comment:
should I add a checkbox for this into #806 ?

Reply:
Yes, add all the checkboxes that come to your mind. It helps in keeping track of the big epic.

Reply:
done